src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/OOPTagHelperResolver.cs

```diff
@@ -141,21 +153,7 @@ protected virtual Task<TagHelperResolutionResult> ResolveTagHelpersInProcessAsyn
 
         private static JObject Serialize(ProjectSnapshot snapshot)
```

Comment:
I would just nuke the serialize method since you no longer have the deserialize method. It's a one-liner anyhow